action.yml

```diff
@@ -64,6 +68,7 @@ runs:
             BackportTargetBranch = "$backportTargetBranch";
             BackportPRNumber = "$backportPRNumber";
             BackportHeadSHA = "$($response.head.sha)"
+            UseFork = "${{ inputs.use_fork }}"
```

Comment:
`UseFork = "${{ inputs.use_fork }}" -eq "true"` so ConvertTo-Json properly casts it as a boolean :)